README.md

```diff
@@ -19,7 +19,7 @@ The follow shortcodes are now available for your use within the content field:
 - Facebook `[facebook url="https://www.facebook.com/willpd/posts/1001217146572688"]`
 - iFrames (requires code-level configuration of accepted domains) `[iframe src="http://www.buzzfeed.com"]`
 - Infogram `[infogram url="http://infogr.am/washington_marijuana_sales"]`
-- PDF's (requires PDF served from domain with `Access-Control-Allow-Origin` header) `[pdf url="https://assets.fusion.net/edit/pdfs/the_interview_budget_excerpts.pdf"]`
+- PDF's (requires PDF be served locally or from domain with `Access-Control-Allow-Origin` header) `[pdf url="https://assets.fusion.net/edit/pdfs/the_interview_budget_excerpts.pdf"]`
```

Comment:
Minor note: This README file is built from the `readme.txt` file (using the `grunt readme` task), so this change should be made there as well.